extensions/extensions-helm/extensions-helm-spark-infrastructure/aissemble-thrift-server-chart/templates/ingress.yaml

```diff
@@ -0,0 +1,28 @@
+{{ if and .Values.enable .Values.ingress.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ .Release.Name }}-sts-ingress
+  {{- with .Values.ingress.metadata.annotations }}
+  annotations:
+  {{- toYaml . | nindent 8 }}
+  {{- end }}
+spec:
+  ingressClassName: {{ .Values.ingress.ingressClassName }}
+  rules:
+    {{- $releasename := printf "%s-service" .Release.Name }}
+    {{- range .Values.ingress.hosts }}
+    - host: {{ .host | quote }}
+      http:
+        paths:
+          {{- range .paths }}
+          - path: {{ .path }}
+            pathType: {{ .pathType }}
+            backend:
+              service:
+                name: {{ .backend.service.name | default $releasename }}
+                port:
+                  number: {{ .backend.service.port.number }}
```

Comment:
S: We know the default service name and port. We should just use the values already set on `Values.service`.  If someone actually tried to change this to a different value, I'd argue that'd be inherently incorrect.  I could see allowing them to _extend_ with extra services, but not to replace the actual service definition which we're creating in this chart.

Reply:
Could have a static path definition for the service provided by this chart, then allow them to add extra with a `.otherPaths` config.

Reply:
On further examination, I would actually be inclined to take this whole section out and replace it with:

```
{{ .paths | toYaml | nindent 10 }}
```

or something to that effect